### DISCUSSION TITLE
如何使用原生控件

### DISCUSSION THREAD

**Discussion by yanglei629** [HandyOrg/HandyControl#1146] [Q&A] [answered]
handycontrol中的GroupBox控件不是我想要的效果，我如何能够使用wpf原生的GroupBox组件

**Comment by Yopler** [comment, marked as answer]
Hi 🖐,

Actually you use the default WPF `GroupBox`. 
However, HandyControl apply a style to the control. You can just say to your `GroupBox `to have no style (= wpf default style) by writting : 
```xaml
<GroupBox Header="Header" Style="{x:Null}">
            
</GroupBox>
```

![image](https://user-images.githubusercontent.com/45144437/161024310-abf1c874-ff35-43e8-9f2a-a1ecc89823e0.png)

Hope it can help 😊

**Comment by CodingOctocat** [comment]
顺便说一下，如果要覆盖原生控件样式，可以这样 `BasedOn="{StaticResource {x:Type SomeControl}}`
